configs/milvus.yaml

```diff
@@ -16,424 +16,381 @@
 
 # Related configuration of etcd, used to store Milvus metadata & service discovery.
 etcd:
-  endpoints:
-    - localhost:2379
-  rootPath: by-dev # The root path where data is stored in etcd
-  metaSubPath: meta # metaRootPath = rootPath + '/' + metaSubPath
+  data:
+    dir: default.etcd # Embedded Etcd only. please adjust in embedded Milvus: /tmp/milvus/etcdData/
+  endpoints: localhost:2379
   kvSubPath: kv # kvRootPath = rootPath + '/' + kvSubPath
   log:
+    level: info # Only supports debug, info, warn, error, panic, or fatal. Default 'info'.
     # path is one of:
     #  - "default" as os.Stderr,
     #  - "stderr" as os.Stderr,
     #  - "stdout" as os.Stdout,
     #  - file path to append server logs to.
     # please adjust in embedded Milvus: /tmp/milvus/logs/etcd.log
     path: stdout
-    level: info # Only supports debug, info, warn, error, panic, or fatal. Default 'info'.
-  use:
-    # please adjust in embedded Milvus: true
-    embed: false # Whether to enable embedded Etcd (an in-process EtcdServer).
-  data:
-    # Embedded Etcd only.
-    # please adjust in embedded Milvus: /tmp/milvus/etcdData/
-    dir: default.etcd
+  metaSubPath: meta # metaRootPath = rootPath + '/' + metaSubPath
```

Comment:
The fields should be ordered, some fields should be put together, for example, the `xxxPath` fields

Reply:
I will try it